projects/addon-mobile/components/mobile-calendar/mobile-calendar.component.ts

```diff
@@ -139,7 +140,7 @@ export class TuiMobileCalendarComponent implements AfterViewInit {
 
     constructor() {
         inject(TUI_VALUE_STREAM)
-            .pipe(takeUntil(this.destroy$))
+            .pipe(takeUntilDestroyed(this.destroy$))
```

Comment:
Could it be just `takeUntilDestroyed()` ?

Reply:
fixed

Reply:
fixed